src/popup/interface.js

```diff
@@ -234,7 +234,8 @@ function search(searchQuery) {
 
     if (overrideDefaultSearchOnce && this.settings.authRequested && authUrl) {
         const authUrlInfo = new BrowserpassURL(authUrl);
-        this.results = helpers.filterSortLogins(this.logins, authUrlInfo.domain, true);
+        searchQuery = authUrlInfo?.validDomain ? authUrlInfo.domain : authUrlInfo?.host || "";
```

Comment:
```suggestion
        searchQuery = authUrlInfo.validDomain ? authUrlInfo.domain : authUrlInfo.hostname ?? "";
```

Please correct me if I'm wrong, but since we pass `domain` here, domain doesn't include the port number, so for consistency sake we should fallback to `hostname` and not `host` to also not include the port number.

We do have this in the README, I hope it still holds true :sweat_smile: 

> Browserpass can also distinguish credentials meant for different ports, so for example an entry example.com.gpg will show up in Browserpass popup when you browse example.com on any port, however an entry example.com:8080.gpg will only show up on 8080 port.

Other than that, I suppose authUrlInfo can never be null/undefined, so we can drop the `?` right? (and use `??` for minor preference)